src/base/templates/base.hbs

```diff
@@ -77,6 +77,7 @@
   </div><!-- end #main -->
 
   <div id="list-container"></div>
+  <div id="sha-container"></div>
```

Comment:
Unfortunately adding a new element here is the only reasonable way I can think of to add a new page. I think this will get cleaned up once we dissolve AppView.